google-cloud-spanner/src/main/java/com/google/cloud/spanner/DatabaseClientImpl.java

```diff
@@ -106,6 +108,28 @@ public CommitResponse writeAtLeastOnceWithOptions(
     }
   }
 
+  @Override
+  public ServerStream<BatchWriteResponse> batchWriteAtLeastOnce(
+      final Iterable<MutationGroup> mutationGroups) throws SpannerException {
+    return batchWriteAtLeastOnceWithOptions(mutationGroups);
+  }
+
+  @Override
+  public ServerStream<BatchWriteResponse> batchWriteAtLeastOnceWithOptions(
+      final Iterable<MutationGroup> mutationGroups, final TransactionOption... options)
+      throws SpannerException {
+    Span span = tracer.spanBuilder(READ_WRITE_TRANSACTION).startSpan();
+    try (Scope s = tracer.withSpan(span)) {
+      return runWithSessionRetry(
+          session -> session.batchWriteAtLeastOnceWithOptions(mutationGroups, options));
+    } catch (RuntimeException e) {
```

Comment:
Why do we only catch `RuntimeException` here, instead of `Exception`?

Reply:
I'm not 100% sure of the rationale behind it. It's something that is used uniformly across other methods as well (`writeWithOptions`, `writeAtLeastOnceWithOptions`, `runAsync`, etc).

Reply:
Are you aware of the reason?

Reply:
No. So for consistency I agree that we should keep it like this for now.